Pocket.xcodeproj/xcshareddata/xcschemes/Pocket (iOS).xcscheme

```diff
@@ -47,60 +47,60 @@
             skipped = "NO">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "SyncTests"
-               BuildableName = "SyncTests"
-               BlueprintName = "SyncTests"
-               ReferencedContainer = "container:PocketKit">
+               BlueprintIdentifier = "166A81BF2637406C0015AA1D"
```

Comment:
Yes, I made some temporary changes to the scheme and _even though I undid the changes_ Xcode decided to reorder all these test targets. I'm going to take these changes out of the PR since it's just noise